src/app/pdfDownloader/page.tsx

```diff
@@ -12,6 +13,10 @@ const PdfDownloader = async (): Promise<ReactElement> => {
                 {/* @ts-ignore */}
```

Comment:
rebase main, we no longer need a @ts-ignore

Reply:
Done